getting-started/essentials.html.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Fly.io essentials"
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+You really can [get an app up and running in just minutes](https://fly.io/speedrun/) on Fly.io.
+
+But if you want to know a bit more about what's what on the Fly.io platform, then read on. Or skip to the [glossary](#flyio-glossary) for the tl;dr.
+
+Two big concepts to know about how Fly.io works, and one small one: Fly Machines, Fly Launch, and Fly Apps.
+
+## Fly Machines: The basic unit of Doing Stuff on Fly.io
+
+Fly Machines are fast-launching virtual machines (VMs) that are individually runnable and controllable. We build Machines out of the images you give us and we run them where you tell us to.
+
+Machines are the compute on Fly.io,[ our lowest level of orchestration](https://fly.io/blog/carving-the-scheduler-out-of-our-orchestrator/). When you just have to run a VM of a specific size in a specific place, Machines are what you want. If you're thinking in terms of individual VMs, rather than applications comprising groups of VMs, then you're thinking in terms of Machines.
+
+Learn more about [Fly Machines](/docs/machines/).
+
+## Fly Launch: Manage applications built on Fly Machines
+
+Fly Launch is how you manage the whole lifecycle of an application on top of Machines, from starting to scaling to changing and redeploying. If you're not thinking of an individual VM, but rather of an application, like "sandwich-ratings.fly.dev", then you're thinking in terms of Fly Launch.
+
+Fly Launch is our built-from-scratch-for-Fly-Machines orchestrator:
+
+- Create your app and get started fast with framework scanners and useful defaults with the `fly launch` command.
+- Configure your app's deployment and services with the[ fly.toml](/docs/reference/configuration/) configuration file.
+- Perform coordinated deployment of Machines for your app with the `fly deploy` command.
+- Scale your app's Machines[ horizontally](/docs/apps/scale-count/) or[ vertically](/docs/apps/scale-machine/) with the `fly scale` command.
+
+Again, Fly Launch is built on Machines: you can use Fly Launch to manage the scale of your application with a single command, or you can interact directly with Machines for fine-grained control.
+
+Learn more about [Fly Launch](/docs/apps/).
+
+## Fly Apps, Fly Machines, and Fly Launch
+
+A Fly App is just a way to group Machines on the Fly Platform. When we talk about "your app" in how-to docs, we're talking about a Fly App.
+
+You can manage a Fly App's Machines as a group with Fly Launch features. or you can run and manage individual Machines with the Machines API or with `fly machine` flyctl commands. You can even do all of these things in one Fly App. We’re not here to tell you what to build!
+
+## Fly.io glossary
+
+**The Fly.io platform**: Everything Fly.io.
```

Comment:
I'm not sure this needs to be defined, or if it does, a useful definition is probably more detailed?

Reply:
good point!

Reply:
Added a bit more to the definition.